aptos-move/framework/aptos-framework/sources/AptosGovernance.move

```diff
@@ -201,6 +202,13 @@ module AptosFramework::AptosGovernance {
         let voter_address = Signer::address_of(voter);
         assert!(Stake::is_delegated_voter(stake_pool, voter_address), Errors::invalid_argument(ENOT_DELEGATED_VOTER));
 
+        // Voting power does not include pending_active or pending_inactive balances.
+        // In general, the stake pool should not have pending_inactive balance if it still has lockup (requried to vote)
```

Comment:
Sp error at end of sentence.

Should not have -- is this an invariant the stake pool is supposed to satisfy?

Reply:
Yes this is also discussed in a comment on https://github.com/aptos-labs/aptos-core/pull/1860. It makes sense to only count a validator's active stake as voting power. However, the nuance here is whether we allow pending_active and pending_inactive validators who have a sufficient lockup to vote (they might be waiting to join/leave the current epoch's validator set).